NetworkOfPrivateClinics/NetworkOfPrivateClinics/Doctor.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NetworkOfPrivateClinics.CustomExceptions;
+using NetworkOfPrivateClinics.Exceptions;
+
+namespace NetworkOfPrivateClinics
+{
+    public enum DoctorType
+    {
+        Neurologist,
+        Gastroenterologist,
+        Cardiology,
+        Oncologist,
+        Psychiatrist,
+        Dermatologist,
+        Anasthesiologist
+    }
+
+    public class Doctor
+    {
+        public static int DoctorID { get; private set; }
+        public string Name { get; private set; }
+        public string Surname { get; private set; }
+        public DoctorType Type { get; private set; }
+        public double CostOfAdmission { get; private set; }
+        public Dictionary<TimeOnly, Patient> ListOfAppointments { get; set; }
```

Comment:
I dislike this approach because:
1. We cannot distinguish between different dates because we use timeonly. A doctor could have appointments today, tomorrow, in a month, etc.
2. We don't know the duration of appointment in this approach, though it might be useful for us.

I recommend create a separate class for appointment and fix the mistakes above